.github/docs/FileAttachmentsFeature.md

```diff
@@ -0,0 +1,51 @@
+# File Attachments
+
+> ‼️ MUST ENABLE this feature for [your Connect instance]([https://docs.aws.amazon.com/connect/latest/adminguide/message-receipts.html](https://docs.aws.amazon.com/connect/latest/adminguide/enable-attachments.html))
+
+Allow customers and agents to share files using chat. Refer to Admin Guide documentation to [enable attachments for you Connect instance](https://docs.aws.amazon.com/connect/latest/adminguide/enable-attachments.html)
+
+<img width="222" alt="Chat Attachments Feature" src="/.github/screenshots/chat-attachments-screenshot.png">
+
+## Reference
+
+ - Initial release date: 12/21/2020
+ - [Admin Guide Documentation](TODO)
```

Comment:
Is there a follow up task here?  If not, I would just remove this line if it doesn't point to anything.

Reply:
Good catch, forgot to paste the link.